### PR TITLE
Show Location on the Map

### PR DESCRIPTION
**Pull request by evulins**
On hovering over a data point on the graph, the relevant point should be highlighted on the map. This should work for fixed and mobile sessions.

[Trello task](https://trello.com/c/QZoNmMV7/1832-session-details-mobile-fixed-show-location-on-the-map)

### CODE REVIEW

**Review by bazelajoanna — APPROVED** [review]
👏 Works great!

File: app/javascript/react/components/Map/Markers/StreamMarkers.tsx [hunk 1]
```diff
@@ -20,6 +23,8 @@ const StreamMarkers = ({ sessions, unitSymbol }: Props) => {
     {}
   );
   const polylineRef = useRef<google.maps.Polyline | null>(null);
+  const hoverStreamId = useSelector(selectHoverStreamId);
```
Comment (JoannaThelema) [inline app/javascript/react/components/Map/Markers/StreamMarkers.tsx:26]: do we need `hoverStreamId` here?

**Review by JoannaThelema — APPROVED** [review]
🙌

File: app/javascript/react/components/Map/Markers/StreamMarkers.tsx (hunk 1 above)
Comment (evulins) [reply app/javascript/react/components/Map/Markers/StreamMarkers.tsx:26]: No :)

**Review by JoannaThelema — APPROVED** [review]
👍

**Pull request merged**